static/app/actionCreators/indicator.tsx

```diff
@@ -116,7 +116,7 @@ const prettyFormString = (val: ChangeValue, model: FormModel, fieldName: string)
     return PRETTY_VALUES.get(val);
   }
 
-  return `${val}`;
+  return val;
```

Comment:
@priscilawebdev would this be better? `val` can be either `FieldValue` or `Record<string, any>`.

```suggestion
  return typeof value === 'object' ? val : String(val);
```